packages/labs/ssr/src/lib/module-loader.ts

```diff
@@ -11,6 +11,8 @@ import * as vm from 'vm';
 import resolveAsync from 'resolve';
 import {builtinModules} from 'module';
 
+import './typings/node.js';
```

Comment:
I think we could define our own type for use in the public APIs (only `ModuleRecord`?) where we now have references to `vm.Module`.

Reply:
Added a new `VmModule` type for public APIs.